src/main/java/tetris/Tetromino.java

```diff
@@ -18,16 +19,7 @@
     private List<Square> miniShape = new ArrayList<>();
 
     public Tetromino() {
-        createOShape();
-        createIShape();
-        createJShape();
-        createLShape();
-        createSShape();
-        createZShape();
-        createTShape();
-        createPShape();
-        createUShape();
-        createMiniShape();
+        getRandomShape();
```

Comment:
I don't think this should be here. Where does the shape go when Tetromino is instantiated?

Reply:
oh right meant to change that. We want to create the shape that getRandomShape will get right?